app/component/SummaryPage.js

```diff
@@ -93,6 +99,26 @@ export function reportError(error) {
   });
 }
 
+const startClient = context => {
+  const { realTime } = context.config;
+  let agency;
```

Comment:
Preferably call it feedId rather than agency because within a feed there can be multiple agencies and these realtime feeds cover data for multiple agencies within a feed.